pkg/lakom/resolvetag/admission.go

```diff
@@ -199,7 +199,7 @@ func (h *handler) handlePod(ctx context.Context, p *corev1.Pod, logger logr.Logg
 	return nil
 }
 
-func (h *handler) handleContainer(ctx context.Context, containerName, image string, kcr utils.KeyChainReader, logger logr.Logger) (string, error) {
+func (h *handler) handleContainer(ctx context.Context, _, image string, kcr utils.KeyChainReader, logger logr.Logger) (string, error) {
```

Comment:
You can completely remove the `containerName` arg.
```suggestion
func (h *handler) handleContainer(ctx context.Context, image string, kcr utils.KeyChainReader, logger logr.Logger) (string, error) {
```